### DISCUSSION TITLE
Laravel Backup to Dropbox Backup failed because Unable to write file at location

### DISCUSSION THREAD

**Discussion by Jnyawach** [spatie/laravel-backup#1524] [Q&A] [answered]
Hello I am trying to backup laravel to dropbox. I keep getting this error: Exception message: rmdir(C:\xampp\htdocs\careermove\storage\app/backup-temp\temp): Directory not empty

Exception trace: #0 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(231): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'rmdir(C:\xampp...', 'C:\xampp\htdocs...', 173) #1 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap{closure}(2, 'rmdir(C:\xampp...', 'C:\xampp\htdocs...', 173) #2 C:\xampp\htdocs\careermove\vendor\spatie\temporary-directory\src\TemporaryDirectory.php(173): rmdir('C:\xampp\htdocs...') #3 C:\xampp\htdocs\careermove\vendor\spatie\temporary-directory\src\TemporaryDirectory.php(99): Spatie\TemporaryDirectory\TemporaryDirectory->deleteDirectory('C:\xampp\htdocs...') #4 C:\xampp\htdocs\careermove\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(174): Spatie\TemporaryDirectory\TemporaryDirectory->delete() #5 C:\xampp\htdocs\careermove\vendor\spatie\laravel-backup\src\Commands\BackupCommand.php(58): Spatie\Backup\Tasks\Backup\BackupJob->run() #6 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(36): Spatie\Backup\Commands\BackupCommand->handle() #7 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Container\Util.php(41): Illuminate\Container\BoundMethod::Illuminate\Container{closure}() #8 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(93): Illuminate\Container\Util::unwrapIfClosure(Object(Closure)) #9 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(37): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Array, Object(Closure)) #10 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Container\Container.php(653): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Array, Array, NULL) #11 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Console\Command.php(171): Illuminate\Container\Container->call(Array) #12 C:\xampp\htdocs\careermove\vendor\symfony\console\Command\Command.php(291): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle)) #13 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Console\Command.php(156): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle)) #14 C:\xampp\htdocs\careermove\vendor\spatie\laravel-backup\src\Commands\BaseCommand.php(28): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #15 C:\xampp\htdocs\careermove\vendor\symfony\console\Application.php(989): Spatie\Backup\Commands\BaseCommand->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #16 C:\xampp\htdocs\careermove\vendor\symfony\console\Application.php(299): Symfony\Component\Console\Application->doRunCommand(Object(Spatie\Backup\Commands\BackupCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #17 C:\xampp\htdocs\careermove\vendor\symfony\console\Application.php(171): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #18 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Console\Application.php(102): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #19 C:\xampp\htdocs\careermove\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(129): Illuminate\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #20 C:\xampp\htdocs\careermove\artisan(37): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput)) #21

How can I solve this? My Dropbox permission are updated and okay. Please help

**Comment by erikn69** [comment, marked as answer]
This package has no windows support

**Reply by Jnyawach** [reply]
@erikn69 I still face the same issue even after deploying to digital ocean

**Reply by PaolaRuby** [reply]
>20 days ago

feedback after 20 days, don't wait any answers 😄

**Reply by Jnyawach** [reply]
Hello @PaolaRuby , Do you know the workaround?

**Reply by PaolaRuby** [reply]
There is no support for dropbox, if you find the solution post it here

**Reply by erikn69** [reply]
Maybe another option [flysystem-google-drive](https://github.com/masbug/flysystem-google-drive-ext), 
folder permissions?
try to delete manually that folder

**Reply by Jnyawach** [reply]
Thank you.  I realized where the problem was and fixed it

**Reply by Jnyawach** [reply]
The app name in the config.backup and .env were not matchin
